### DISCUSSION TITLE
dark mode

### DISCUSSION THREAD

**Discussion by nofonbg** [tbnobody/OpenDTU#581] [Show and tell]
falls Energiesparen ein Thema ist

--bs-body-bg: #000;
--bs-card-bg: #2d2d2d;;
--bs-body-color: #d5d5d5;

in webapp_dist/js/app.js.gz 

![dark_mode](https://user-images.githubusercontent.com/118652526/218304748-423dd6cb-5938-4cbf-99df-6df2fc4cc9b7.jpg)

**Comment by gcPhil** [comment]
Sieht gut aus!
**Ich fände es praktisch wenn man das einfach über die DTU-Einstellungen aktivieren könnte, statt es bei jedem Update neu ändern zu müssen.**

**Comment by gcPhil** [comment]
Ließe sich noch auf Modals, Menüs und Accordion erweitern mit:

--bs-modal-bg: #2d2d2d;

--bs-dropdown-bg: #2d2d2d;
--bs-dropdown-link-color: #d5d5d5;
--bs-dropdown-link-hover-bg: #6c757d;
--bs-dropdown-link-hover-color: #d5d5d5;

--bs-accordion-bg: #2d2d2d;
--bs-accordion-color: #ededf0;
--bs-accordion-btn-color: #ededf0;

**Comment by nofonbg** [comment]
Dazu noch die Dropdownliste für das Sprachpaket:
.me-2{background-color: #767676!important}

Das greif leider nicht auf die Variablen zurück, muss imho bei -me-2 ergänzt werden?

**Comment by gcPhil** [comment]
Laut Issue https://github.com/tbnobody/OpenDTU/issues/623 ist geplant einen Dark Mode zu implementieren nachdem Bootstrap 5.3 final verfügbar ist.

**Reply by nofonbg** [reply]
👍🏼

**Comment by mathieucarbou** [comment]
thanks a lot for the dark mode, I love it!
